### DISCUSSION TITLE
Counter1. Tele

### DISCUSSION THREAD

**Discussion by deffel6** [arendst/Tasmota#13390] [Q&A] [answered]
Hello !
Is it possible to set a Rule o. something this :
Counter1 measuring a time, and send a Tele with this time, next Tele again than a new Time measuring?

thanks a Lot

**Comment by barbudor** [comment, marked as answer]
If you set a `CounterType 1` then the counter reports the average period of the pulse in seconds
WIth `Setoption79 1`, the average is computed at each teleperiod

**Comment by deffel6** [comment]
yes, I try to count the Puls chance, so I need the Time only it chance,  
Exemple : now I have a Puls time von 120 Second, if tele period 60, I got this time twice (Mqtt) , what I try is,  120S Pulstime. one Tele, if Pulsetime 180 Second one Tele again.  The reason is, I try to count the Chance additional

**Reply by barbudor** [reply]
Sorry, but I don't undersand what you are writting
